internal/kafka/internal/clusters/standalone_provider.go

```diff
@@ -583,3 +583,7 @@ func (s *StandaloneProvider) GetClusterResourceQuotaCosts() ([]types.QuotaCost,
 	var quotaCostList []types.QuotaCost
 	return quotaCostList, nil
 }
+
+func (s *StandaloneProvider) CheckIfOrganizationIsTheClusterOwner(externalOrganizationID, clusterID, clusterExternalID string) error {
+	return nil
```

Comment:
Shouldn't this check the config file?

Reply:
Which config? The cluster list config or the quota list management configuration?

Reply:
I was thinking about the quota list management configuration

Reply:
Right. I think the list config is specific to a Kafka quota i.e we do not data plane cluster quota info in that config.